tests/JMS/Serializer/Tests/Serializer/XmlSerializationTest.php

```diff
@@ -89,7 +89,7 @@ public function testPropertyIsCollectionOfObjectsWithAttributeAndValue()
 
     /**
      * @expectedException JMS\Serializer\Exception\InvalidArgumentException
-     * @expectedExceptionMessage The document type "<!DOCTYPE author [<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">]>" is not allowed. If it is safe, you may add it to the whitelist configuration.
+     * @expectedExceptionMessage The document type "<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">" is not allowed. If it is safe, you may add it to the whitelist configuration.
```

Comment:
Do we have a test for whitelisting doc types, but then using a doc type that is not whitelisted?